ci/build.sh

```diff
@@ -5,3 +5,30 @@ set -e
 PROJECT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )/.."
 
 "$PROJECT_DIR"/gradlew --no-daemon --info --stacktrace clean build
+
+BINTRAY_USER="$BINTRAY_USER"
+BINTRAY_API_KEY="$BINTRAY_API_KEY"
+
+GIT_BRANCH=`git rev-parse --abbrev-ref HEAD`
```

Comment:
uh, Travis checks out commit directly so we can't ask git for current branch

I'll workaround that in a way that it'll work both locally and on Travis